inception-project/inception
# #1222 - New page for simulating evaluation

**What's in the PR**
* Evaluation Simulation Page in the project menu

**How to test manually**
* Open Project
* Click on the Evaluation Simulation Menu Option
* Select a recommender. Press Start
* The learning Curve should show up

**Automatic testing**

**Documentation**
